crates/core/src/lib.rs

```diff
@@ -33,7 +33,10 @@ pub mod prelude {
     pub use diesel::{
         dsl::{any, exists, not},
         expression_methods::*,
+        pg::Pg,
         query_dsl::{BelongingToDsl, GroupByDsl, JoinOnDsl, QueryDsl, RunQueryDsl, SaveChangesDsl},
+        sql_types::Bool,
```

Comment:
I think it might be good to just re-export all of sql_types here to reduce future changes

Reply:
Isn't this that. What needs to change?

Reply:
Actually not using these. I'll remove.

Reply:
haha sounds good.  it would have been something like replacing sql_types::Bool with just sql_types in case we needed other items from that module in the future